zebROS_ws/src/ctre_interfaces/include/ctre_interfaces/candle_command_interface.h

```diff
@@ -0,0 +1,83 @@
+#pragma once
+
+#include "ctre_interfaces/candle_state_interface.h"
+#include "state_handle/command_handle.h"
+#include <vector>
+#include <optional>
+
+using namespace std;
```

Comment:
Don't use `using namespace` inside header files - https://stackoverflow.com/questions/14575799/using-namespace-std-in-a-header-file

Reply:
wasn't aware of this issue, should be fixed now

Reply:
Yeah, one of many fun c++ pitfalls.